components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/edit-user-roles-finish.jsp

```diff
@@ -125,16 +125,17 @@
             </script>
 <%
         }
+
     } catch (Exception e) {
-         String message = MessageFormat.format(resourceBundle.getString("role.list.cannot.update"), Util.decodeHTMLCharacters(username), e.getMessage());
-         CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request);
+        	String message = MessageFormat.format(resourceBundle.getString("role.list.cannot.update"), Util.decodeHTMLCharacters(username), e.getMessage());
+        	CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request);
         if(viewUsers){
 %>
             <script type="text/javascript">
                 location.href = "view-roles.jsp?username=<%=URLEncoder.encode(username)%>";
             </script>
 <%
-        } else {
+} else {
 %>
```

Comment:
Seems there is only code formatting changes here. Don't include code formatting changes with bug fixes because it clutters the fix.